packages/plot/src/utils.js

```diff
@@ -138,3 +138,29 @@ export const isDomainRangeEqual = (graphProps, nextGraphProps) => {
     isEqual(graphProps.range, nextGraphProps.range)
   );
 };
+
+export const findLongestWord = label => {
+  var longestWord = (label || '').split(' ').sort(function(a, b) {
+    return b.length - a.length;
+  });
+  return longestWord[0].length;
+};
+
+export const amountToIncreaseWidth = longestWord => {
+  switch (true) {
+    case longestWord < 3:
```

Comment:
I would rather replace this with a formula. (multiplying the nr of characters with a constant)

Reply:
done